Author: Cyril Roelandt
Bug-Debian: https://bugs.debian.org/671545
Description: Fix FTBFS on hurd-i386
 Fix hurd-i386 because of the unconditional use of PATH_MAX in gphoto2/main.c
Reviewed-by: Herbert Parentes Fortes Neto <hpfn@ig.com.br>
Last-Update: 2014-12-22

--- a/gphoto2/main.c
+++ b/gphoto2/main.c
@@ -2221,7 +2221,7 @@
 		}	
 	}
 	if (!gp_params.hook_script) {
-		char buf[PATH_MAX];
+		char buf[256];
 		if (gp_setting_get("gphoto2","hook-script",buf)>=0) {
 			gp_params.hook_script = strdup(buf);
 			/* Run init hook */
